### DISCUSSION TITLE
Propose to change the project name Hypercrx -> HyperCRX

### DISCUSSION THREAD

**Discussion by xiaoya-yaya** [hypertrons/hypertrons-crx#772] [Ideas]
Hypercrx as the tool demo paper was recently accepted by ICPC 2024 conference and have received quite good review comments!

According to the reviewers' comments, and in conjunction with the community feedback, HyperCRX as the project name will be more easily pronounced for newcomers. Thus, we propose to change the project name from Hypercrx to HyperCRX.

<img width="1328" alt="image" src="https://github.com/hypertrons/hypertrons-crx/assets/56542972/340524e8-a58b-4561-b26d-763a59b6dc95">

**Comment by tyn1998** [comment]
Agree +1

**Comment by will-ww** [comment]
Agree +1
